dbms/src/Server/Server.cpp

```diff
@@ -537,11 +538,13 @@ std::unique_ptr<std::thread> initStores(Context & global_context, const LoggerPt
         {
             wait_proxy(tiflash_instance_wrap);
         }
+        ThreadPool thread_pool(SettingMaxThreads().getAutoValue());
         auto storages = global_context.getTMTContext().getStorages().getAllStorage();
-        int init_cnt = 0;
-        int err_cnt = 0;
-        for (auto & [table_id, storage] : storages)
-        {
+
```

Comment:
this part of code are move to a standalone file in this PRhttps://github.com/pingcap/tiflash/pull/7060/files#diff-00c4c381cdac70e08aa0bec334e90ea23653d8bc1a92c33b03a65bd928ce6a87R41-R47